schedule/kernel/install_ltp_uefi_baremetal.yaml

```diff
@@ -1,14 +1,17 @@
 name:          UEFI_baremetal_basic
```

Comment:
Not related to PR, but wouldn't be better if it was "prepare_uefi_baremetal_ltp" ?

Reply:
I selfishly wanted to have install_ltp in the name (it doesn't have to be in the setup), so people find that :(.